src/WarcraftLegacies.Source/Quests/Naga/QuestLostOnes.cs

```diff
@@ -39,13 +39,15 @@ public sealed class QuestLostOnes : QuestData
     /// <inheritdoc/>
     /// </summary>
     protected override string CompletionPopup =>
-      "The Lost Ones tribe has joined our cause";
+      "The Draenai of the Ashtongue tribe have joined our cause!";
 
     /// <summary>
     /// <inheritdoc/>
     /// </summary>
     protected override string RewardDescription =>
-      "Control of all units in the Draenei camp, Enable to Build the Draenei Hut and train Akama";
+      "Gain control of the Draenei camp.\n" +
```

Comment:
It probably makes sense to have these on separate lines, but since we don't use this technique for any quest currently could we have it as a single-sentence list?